collect_app/src/main/res/layout/annotate_widget.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeButton
+        android:id="@+id/capture_button"
+        style="?materialButtonStyle"
```

Comment:
This (and the other buttons in this PR) should use `?widgetButtonQuestionWidgetStyle` I think?